packages/_vue3-migration-test/src/x-modules/search/components/test-redirection.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
I can't make this feature work on my local machine, as the search request is not executed.

Reply:
In my local machine is working. What Vuex version do you have? Take into account this comment of @joseacabaneros https://github.com/empathyco/x/pull/1497#discussion_r1635139691

Reply:
Sorry, yes it was it. Now it works, thanks.